dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/EnrollmentAnalyticsController.java

```diff
@@ -93,7 +93,7 @@
     @NotNull
     private DimensionMapperService dimensionMapperService;
 
-    @PreAuthorize( "hasRole('F_PERFORM_ANALYTICS_EXPLAIN')" )
+    @PreAuthorize( "hasRole('ALL') or hasRole('F_PERFORM_ANALYTICS_EXPLAIN')" )
```

Comment:
Should we do this also for `AnalyticsController`?

Reply:
oh yes, sorry, done